map/local_data.html

```diff
@@ -74,20 +112,40 @@ <h1>BCU Labs - Data Team</h1>
                 'line-color': [
                     'match',
                     ['get', 'LTS'],
-                    0, 'black',
+                    0, colors[0],
                     // Colors based on 5 equally spaced from 'turbo' colormap
-                    1, '#28BCEB',
-                    2, '#A4FC3C',
-                    3, '#FB7E21',
-                    4, '#7A0403',
-                    'white'
+                    1, colors[1],
+                    2, colors[2],
+                    3, colors[3],
+                    4, colors[4],
+                    colors[5]
```

Comment:
I noticed that this line breaks the expected format of `5, colors[5]` but I am not familiar enough with the code to know why it is like this. @sckilcoyne do you know the reason why the last line does not have an index?

Reply:
I assumed it was a catch all, anything that wasn't 1-4 would be the final color. Not sure if that is actually how that would work though.